actions/filter-files/index.js

```diff
@@ -64,18 +64,41 @@ module.exports = ({
     globsRaw,
     inputFiles,
     invert,
+    conjunctive,
     core,
 }) => {
-    const extensions = parseList(extensionsRaw);
-    const exactFiles = parseList(exactFilesRaw);
-    const globMatcher = picomatch(parseList(globsRaw));
-    const directories = exactFiles.filter((name) => name.endsWith("/"));
+    const filters = [];
+    if (exactFilesRaw) {
```

Comment:
In order for `every` to work, we should only include filters that have been specified.